README.md

```diff
@@ -1,6 +1,16 @@
 Chitter Challenge
 =================
```

Comment:
Try running rubocop by just typing "rubocop" to see where you have 'offenses' in your code